sdk/search/Microsoft.Azure.Search/tests/Tests/FieldBuilderTests.cs

```diff
@@ -416,11 +417,10 @@ public void FieldBuilderFailsWithHelpfulErrorMessageOnUnsupportedPropertyTypes(T
             string expectedErrorMessage =
                 $"Property '{invalidPropertyName}' is of type '{modelType.GetProperty(invalidPropertyName).PropertyType}', " +
                 "which does not map to an Azure Search data type. Please use a supported data type or mark the property with " +
-                "[JsonIgnore] or [FieldBuilderIgnore] and define the field by creating a Field object." +
-                $"{Environment.NewLine}Parameter name: {nameof(modelType)}";
```

Comment:
I would like to know why all these were changed. I intentionally used the same format as ArgumentExceptions.